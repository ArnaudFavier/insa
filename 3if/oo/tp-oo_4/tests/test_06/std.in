PC poly1 3 4 4 2 2 1
PC poly2 1 1 -1 2 -3 0 -1 -3 1 0
PC poly3 1 1 2 3
HIT poly3 1 1
LIST
MOVE poly1 -2 -1
LIST
HIT poly1 1 3
HIT poly1 2 1
HIT poly1 0 0
HIT poly1 3 2
HIT poly1 2 2
HIT poly2 -3 0
HIT poly2 1 0
HIT poly2 -1 -2
HIT poly2 -2 0
HIT poly2 1 2
HIT poly2 -2 -2
EXIT
