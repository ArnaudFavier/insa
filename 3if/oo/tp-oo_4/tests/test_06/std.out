OK
OK
ERR
NO
poly1 C [(3;4) (4;2) (2;1)]
poly2 C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
OK
poly1 C [(1;3) (2;1) (0;0)]
poly2 C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
YES
YES
YES
NO
NO
YES
YES
YES
YES
NO
NO
